docs/MODULES.md

```diff
@@ -154,6 +156,18 @@ where `options` is an object with the following properties:
 * offset: step of image with which current image is to be blended(Two steps back is -2, three                steps back is -3 etc; default -2) 
 * func: function used to blend two images (default : function(r1, g1, b1, a1, r2, g2, b2, a2) {            return [ r1, g2, b2, a2 ] })
 
+## Blob Analysis
+
+This module uses Opencv.js for detecting and marking blob/region in microscopic images. It requires an opencv.js file to 
+be loaded before using the functionalities which is currently being loaded to the webpage via script.
```

Comment:
Great - should we say whether it works on node or not, and what will shortly be changing? Otherwise this looks good!

Reply:
Yeah it would work in node just need to make some changes there, will probably push them tomorrow! Thanks for your patience.